pyproject.toml

```diff
@@ -66,6 +66,7 @@ dependencies = [
     "requests>=2.32",
     "ruamel.yaml>=0.17.0",
     "scipy>=1.13.0",
+    "scipy>=1.14.1; platform_system == 'Windows'",
```

Comment:
@njzjz can you add a comment linking to https://github.com/scipy/scipy/issues/21052 to explain the platform-dependent version pin?

Reply:
added in d6a0b5aed6825c969f6370f5b3f9119520d6d20b